Sources/App/Models/Release.swift

```diff
@@ -35,3 +35,9 @@ extension Release {
         url = node.url
     }
 }
+
+enum SignificantReleaseKind: String {
```

Comment:
This didn't feel like it deserved its own file. Here seems like a reasonable place for it to live.